### PR TITLE
feature/CVSB-194

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/app/app.scss [hunk 1]
```diff
@@ -206,7 +206,7 @@ ion-item-group {
 ion-list.vehicle-list{
   border-bottom: 0.5px solid $list-ios-border-color;
   margin-bottom: 10px;
-  background: #fff; 
+  background: #fff;
```
Comment (acozianu) [inline src/app/app.scss]: Please use mixins

File: src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+page-reasons-selection {
+  .list-header-ios {
+    font-size: 1rem;
```
Comment (acozianu) [inline src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss]: I think we are using px as a units of measure

File: src/pages/testing/test-abandonment/test-abandoning/test-abandoning.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  .content-ios {
+    background-color: $content-background-color;
+  }
+
+  .list-ios {
+    margin-bottom: 0;
+  }
+
+  .list-header-ios {
+    font-size: 1rem;
```
Comment (acozianu) [inline src/pages/testing/test-abandonment/test-abandoning/test-abandoning.scss]: Same here

File: src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    text-transform: none;
+    background-color: $list-ios-header-background-color;
+  }
+
+  .item-ios {
+    padding-left: 1.5rem;
```
Comment (acozianu) [inline src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss]: Same here

File: src/pages/testing/test-abandonment/test-abandoning/test-abandoning.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+  .item-ios {
+    padding-left: 1.5rem;
+  }
+
+  .row {
+    background-color: $list-ios-header-background-color;
+
+    .label-ios {
+      color: #333;
```
Comment (acozianu) [inline src/pages/testing/test-abandonment/test-abandoning/test-abandoning.scss]: Please use mixins

File: src/pages/testing/test-creation/test-create/test-create.scss [hunk 6]
```diff
@@ -1,50 +1,54 @@
...
+        }
+      }
+
+      .btn-add-test {
+        font-size: 14px;
+        color: color($colors, primary);
+      }
+
+      .item-ios.bottom-divider {
+        border-top: 0.55px solid #c8c7cc;
```
Comment (acozianu) [inline src/pages/testing/test-creation/test-create/test-create.scss]: Use mixins

File: src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss (hunk 2 above)
Comment (vladddragomir) [reply src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss]: done

File: src/pages/testing/test-abandonment/test-abandoning/test-abandoning.scss (hunk 3 above)
Comment (vladddragomir) [reply src/pages/testing/test-abandonment/test-abandoning/test-abandoning.scss]: done

File: src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss (hunk 4 above)
Comment (vladddragomir) [reply src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss]: done

File: src/pages/testing/test-abandonment/test-abandoning/test-abandoning.scss (hunk 5 above)
Comment (vladddragomir) [reply src/pages/testing/test-abandonment/test-abandoning/test-abandoning.scss]: done

File: src/app/app.scss [hunk 7]
```diff
@@ -278,6 +278,9 @@ ion-list.vehicle-list{
         font-size: $font-size-medium;
       }
     }
-  } 
+  }
 }
 
+.danger-action-button {
+  color: color($colors, danger) !important;
```
[the remark this answers is not in the document]
Comment (vladddragomir) [reply src/app/app.scss]: in order to override an alert's action button text colour

File: src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (vladddragomir) [reply src/pages/testing/test-abandonment/reasons-selection/reasons-selection.scss]: done

File: src/pages/testing/test-creation/test-create/test-create.scss (hunk 6 above)
Comment (vladddragomir) [reply src/pages/testing/test-creation/test-create/test-create.scss]: done

File: src/app/app.scss (hunk 1 above)
Comment (vladddragomir) [reply src/app/app.scss]: done

File: test-config/data-mocks/test-abandonment-reasons-data.mock.ts [hunk 8]
```diff
@@ -0,0 +1,18 @@
+export class TestAbandonmentReasonsDataMock {
```
Comment (adpetrescu) [inline test-config/data-mocks/test-abandonment-reasons-data.mock.ts]: why is this a mock and should this be on Git?

Comment (adpetrescu) [inline test-config/data-mocks/test-abandonment-reasons-data.mock.ts]: why is this a mock and should this be on Git?

Comment (vladddragomir) [reply test-config/data-mocks/test-abandonment-reasons-data.mock.ts]: done

**Pull request merged**